spec/acceptance/foreman_cli_plugins_spec.rb

```diff
@@ -17,22 +17,24 @@ class { 'foreman::cli':
         if $facts['os']['family'] == 'RedHat' {
           include foreman::cli::ansible
```

Comment:
Ansible exists in Debian too (https://github.com/theforeman/foreman-packaging/tree/deb/develop/dependencies/focal/hammer_cli_foreman_ansible), so no idea why this is in the RH-only section, mind moving it?

Reply:
Looks like it was added in https://github.com/theforeman/puppet-foreman/commit/acdaea2739a3d71c84b58f95af2e5924ae1b9841 (2018) and it was packaged in https://github.com/theforeman/foreman-packaging/commit/96c868bfae79aa9f2c3a9e1ceddaa87a681e23cc (2019). Nobody bothered to check it in the mean time. Will update.